realtime.go

```diff
@@ -361,7 +390,7 @@ func parseVehicle(vehiclePosition *gtfsrt.VehiclePosition, opts *ParseRealtimeOp
 		ID:                parseTripDescriptor(vehiclePosition.Trip, opts),
 		IsEntityInMessage: false,
 	}
-	return trip, vehicle, false
+	return trip, vehicle, true
```

Comment:
It looks like the last field is always true now so we could get rid of the last return value, and at the call site set `ok = true`